docs/protocol.md

```diff
@@ -326,142 +326,107 @@ a subsequent visit) some $passphrase$:
+|                                                                                                                      | $\longrightarrow$ `RequestKeys` |                                                                               |
+|                                                                                                                      |                                 | $`pks = \{(pk_{R,i}^{APKE}, pk_{R,i}^{PKE}, pk_{R,i}^{fetch})\}`$ for all $i$ |
+|                                                                                                                      | $`pks \longleftarrow`$          |                                                                               |
+| **TODO:** verification per $NR$ and $J_i$                                                                            |                                 |                                                                               |
+|                                                                                                                      |                                 |                                                                               |
+| **Reply case:** The journalist replaces their own keys with those of the source to whom they are replying:           |                                 |                                                                               |
+| $`pks \gets pks \setminus \{pk_S^{APKE}, pk_S^{PKE}, pk_S^{fetch}\} \cup \{pk_R^{APKE}, pk_R^{PKE}, pk_R^{fetch}\}`$ |                                 |                                                                               |
+
+### 6. Sender submits a message <!-- Figure 1 as of 7703a58 -->
+
+Then, for some message $m$, for all keys $(pk_{R,i}^{APKE}, pk_{R,i}^{PKE},
+pk_{R,i}^{fetch}) \in pks$:
+
+| Source                                                                              |                                 | Server                                         |
+| ----------------------------------------------------------------------------------- | ------------------------------- | ---------------------------------------------- |
+| $`pt \gets m \Vert pk_S^{fetch} \Vert pk_S^{PKE} `$                                 |                                 |                                                |
+| $`ct^{APKE} \gets \text{SD-APKE.AuthEnc}(sk_S^{APKE}, pk_{R,i}^{APKE}, pt, NR, -)`$ |                                 |                                                |
+| $`ct^{PKE} \gets \text{SD-PKE.Enc}(pk_{R,i}^{PKE}, pk_S^{APKE}, -, -)`$             |                                 |                                                |
+| $`C_S \gets (ct^{APKE}, ct^{PKE})`$                                                 |                                 |                                                |
```

Comment:
I agree this is also what's in the manuscript.  
I don't know if this is the most accurate way of writing it though, so this is a manuscript followup todo/clarification: 

Each of these $(ct^{APKE}, ct^{PKE})$ have subcomponents. 
in $(ct^{APKE}$ , the message is AuthEnc'd, leading to (message shared secret encaps, message ciphertext, and the encapsulation of the psk).

Then the metadata is built, which does SD-PKE.Enc to the things required to decrypt the message: the message shared secret encaps, the psk encaps, and the sender pubkey, leading to a metadata ciphertext and metadata encapsulation.
 
What is finally sent to the server is `(Metadata shared secret encapsulation, Metadata ciphertext (contains message ss encaps, psk encaps, sender pubkey), message ciphertext, X, Z)`.